test/behavior/array.zig

```diff
@@ -58,9 +57,6 @@ test "array dot len const expr" {
     });
 }
 
-const ArrayDotLenConstExpr = struct {
```

Comment:
please check the git history of this and make sure a test that referenced this decl was not accidentally deleted

Reply:
<https://github.com/ziglang/zig/commit/581edd643> first added in its modern form here
<https://github.com/ziglang/zig/commit/0cd361219> moved to `_stage1.zig` here
<https://github.com/ziglang/zig/commit/6d6cf5984> implemented here

Reply:
asserting that `y: [some_array.len]u8` properly turned to `y: [4]u8` is likely what it was meant to test, but never added

Reply:
<https://github.com/ziglang/zig/commit/6b3ce918d> first added ever here
<https://github.com/ziglang/zig/commit/dab3ddab4> getting closer to modern zig